python/pathfinder.py

```diff
@@ -1,15 +1,18 @@
 import heapq
-
 import vim
 from node import Node, StartNode
 from window import cursor_in_same_position
 
 
 class Motion:
     def __init__(self, dict):
-        self.motion = dict["motion"].decode()
+        if vim.eval("has('nvim')"):
```

Comment:
We could just check the type of the value:
```suggestion
        if type(dict["motion"]) == bytes:
```

Reply:
Personally I'd rather check whether dealing with neovim or vim, do that and be done with it. Furthermore I think this would also still have the same problem as my proof of concept, the computation to check whether `dict["motion"]` is of type bytes is done for every invocation of the motion class, instead of just setting a boolean with `has('nvim')` just once and using this in a `if` clause.